recipes/flowutils/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "FlowUtils" %}
+{% set version = "0.9.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FlowUtils-{{ version }}.tar.gz
+  sha256: b745c4fdf3b26be930880745bdfc7ca58b6657b82b8c1ce48a83a1e63a11924b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy >=1.17
```

Comment:
You already have a `pin_compatible` below that will pin at least that high.
```suggestion
```

Reply:
Thanks, fixed. Any pointers on this?

> Seems like FlowKit recipe was the issue so I will open a separate PR for that package. It relies on these 2 packages in it's requirements so let me know if I need to wait for this PR to merge before that could work.